File: RUBY/Cap03/c03ex03.rb
# c03ex03.rb

#!/usr/bin/env ruby

print "Entre quantidade de horas trabalhadas ..: "
ht = gets.chomp.to_f

print "Entre valor do salario-hora ............: "
vh = gets.chomp.to_f

print "Entre valor do percentual de desconto ..: "
pd = gets.chomp.to_f

sb = ht * vh
td = (pd / 100.0) * sb
sl = sb - td

puts "\nSalario Bruto ....: #{sb.round(2)}"
puts "Salario Liquido ..: #{sl.round(2)}"

print "\nTecle <Enter> para encerrar... "
gets

File: PYTHON/Cap03/c03ex03.py
# c03ex03.py

ht = float(input("Entre quantidade de horas trabalhadas ..: "))
vh = float(input("Entre valor do salario-hora ............: "))
pd = float(input("Entre valor do percentual de desconto ..: "))

sb = ht * vh
td = (pd / 100) * sb
sl = sb - td

print("\nSalario Bruto ...:", f"{sb:.2f}")
print("Salario Liquido .:", f"{sl:.2f}")

input("\nTecle <Enter> para encerrar... ")
